proposals/0344-distributed-actor-runtime.md

```diff
@@ -144,80 +143,72 @@ This code snippet showcases what kind of distributed actors one might want to im
 
-> At the time of writing, we–the proposal authors–have released a work in progress [peer-to-peer cluster actor system implementation](https://www.swift.org/blog/distributed-actors/) that is tracking this evolving language feature. It can be viewed as a reference implementation for the language features and `DistributedActorSystem` protocol discussed in this proposal. 
+> At the time of writing, we–the proposal authors–have released a work in progress [peer-to-peer cluster actor system implementation](https://www.swift.org/blog/distributed-actors/) that is tracking this evolving language feature. It can be viewed as a reference implementation for the language features and `DistributedActorSystem` protocol discussed in this proposal.
 >
 
 Below we present the full listing of the `DistributedActorSystem` protocol, and we'll be explaining the specific methods one by one as we go:
 
 ```swift
 // Module: _Distributed
 
-protocol DistributedActorSystem: Sendable { 
-  /// The type of 'ID' assigned to a distributed actor while initializing with this actor system.
-  /// The identity should be meaningfully unique, in the sense that ID equality should mean referring to the
-  /// same 
-  /// 
-  /// The type of 'ID' assigned to a distributed actor while initializing with this actor system.
+protocol DistributedActorSystem: Sendable {
+  /// The type of `ID` assigned to a distributed actor while initializing with this actor system.
   /// The identity should be meaningfully unique, in the sense that ID equality should mean referring to the
-  /// same. 
-  /// 
-  /// A distributed actor created using a specific actor system will use the system's 'ActorID' as 
-  /// the 'ID' type it stores and for its 'Hashable' implementation.
+  /// same distributed actor.
   ///
-  /// ### Implicit distribute actor Codable conformance
-  /// If the 'ActorID' (and therefore also the 'DistributedActor.ID') conforms to 'Codable',
-  /// the 'distributed actor' will gain an automatically synthesized conformance to 'Codable' as well.
+  /// A distributed actor created using a specific actor system will use the system's `ActorID` as
+  /// the `ID` type it stores and for its `Hashable` implementation.
+  ///
+  /// ### Implicit distribute actor `Codable` conformance
+  /// If the `ActorID` (and therefore also the `DistributedActor.ID`) conforms to `Codable`,
+  /// the `distributed actor` will gain an automatically synthesized conformance to `Codable` as well.
   associatedtype ActorID: Sendable & Hashable
-  
-  /// The specific type of the invocation encoder that will be created and populated 
+
+  /// The specific type of the invocation encoder that will be created and populated
   /// with details about the invocation when a remote call is about to be made.
   ///
-  /// The populated instance will be passed to the remoteCall from where it can be
+  /// The populated instance will be passed to the remote call from where it can be
```

Comment:
@xedin Was the callout to `remoteCall` intentional here?

Reply:
Yes, remoteCall reference is intentional here.

Reply:
👌 Reverted

Reply:
That was intentional, thanks for fixing